### PR TITLE
add r-censusgeography via conda_r_skeleton_helper

### PR DESCRIPTION
**Pull request by izahn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by izahn** [comment]
ready for review @conda-forge/help-r

File: recipes/r-censusgeography/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/censusGeography_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/censusGeography/censusGeography_{{ version }}.tar.gz
+  sha256: 2d5044073f36bbf32f1cb934b31644a02ef17a2f97bce0f778e1145a0b688fd9
+
+build:
+  ## dependencies not available on win
+  skip: True  # [win]
```
Comment (dbast) [inline recipes/r-censusgeography/meta.yaml:17]: Please don't skip windows if this is just noarch and the win dependencies are not available.

Change back to 'noarch: generic' (also update the other section + add bld.bat back). We can then ignore that win CI failure and still merge it. The real build + upload in the feedstock runs on Linux and makes the package as it is noarch available everywhere, where the dependencies are available.

Comment (izahn) [reply recipes/r-censusgeography/meta.yaml]: Done, thank you!

File: recipes/r-censusgeography/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/censusGeography_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/censusGeography/censusGeography_{{ version }}.tar.gz
+  sha256: 2d5044073f36bbf32f1cb934b31644a02ef17a2f97bce0f778e1145a0b688fd9
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```
Comment (dbast) [inline recipes/r-censusgeography/meta.yaml:17]:
```suggestion
  number: 0
  noarch: generic
```

**Comment by dbast** [comment]
noarch -> ignoring win failure.

**Pull request merged**